amgut/handlers/open_humans.py

```diff
@@ -75,7 +75,7 @@ class OpenHumansLoginHandler(BaseHandler, OpenHumansMixin):
     _API_URL = urljoin(AMGUT_CONFIG.open_humans_base_url, '/api')
 
     _OAUTH_REDIRECT_URL = urljoin(AMGUT_CONFIG.base_url,
-                                  '/authed/connect/open-humans/')
+                                  'authed/connect/open-humans/')
```

Comment:
(The reason for that one character change is that later arguments `urljoin` will become absolute to a given domain if they contain a forward slash... meaning that:

`urljoin('http://foo.com/abc/', '/def/')` yields `http://foo.com/def/`. This worked in my testing environment because I had no `SITEBASE` set.)